pkg/network/driver/nmstate/integration_test.go

```diff
@@ -0,0 +1,449 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package nmstate_test
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+
+	vishnetlink "github.com/vishvananda/netlink"
+
+	"kubevirt.io/kubevirt/pkg/network/driver/nmstate"
+	"kubevirt.io/kubevirt/pkg/network/driver/procsys"
+	"kubevirt.io/kubevirt/pkg/pointer"
+)
+
+var runIntegrationTests bool
+
+func init() {
+	flag.BoolVar(&runIntegrationTests, "run-integration-tests", false, "run integration tests")
+}
+
+var _ = ginkgo.Describe("NMState", integrationLabel, func() {
+	ginkgo.BeforeEach(func() {
```

Comment:
In this particular case, I think you should follow the code base convention which is to dot import ginkgo/gomega packages since this file now introduce inconsistency.
It also adds noise when reading test files since there are lot of calls to the package functions.

Reply:
I think the dot imports are bad, there are good reasons not to use them and only two reasons to use them (less writing, most other test files in the project use dot imports).

There is a good chance this whole package will move out to an external repo (e.g. in order to use it with binding CNI/s), so keeping it clean is a high priority.

If you search the project, you will see other test files use it like this, so this is surely not the first place in the project and the "convention" has been "broken" long time ago.

IMO if it is consistent in the file and package, it is good enough to be considered consistent.